packages/mobile/components/ArticleContent.tsx

```diff
@@ -0,0 +1,271 @@
+import * as React from "react";
+import { useQuery, useMutation } from "@apollo/react-hooks";
```

Comment:
On a note, you'll want to remove this and the queries imported below, we're going to work with the hooks generated by graphql-codegen